### PR TITLE
DOC: Add graphics to solve_ivp examples

### PR DESCRIPTION
**Pull request by jakobjakobson13**

### CODE REVIEW

File: scipy/integrate/_ivp/ivp.py [hunk 1]
```diff
@@ -398,6 +398,14 @@ def solve_ivp(fun, t_span, y0, method='RK45', t_eval=None, dense_output=False,
       0.05858998 0.02701876]
      [8.         7.5534414  4.25308709 1.73277247 0.7059579  0.287618
       0.11717996 0.05403753]]
+    
+    >>> import matplotlib.pyplot as plt
+    >>> plt.plot(sol.t, sol.y[0], label='y0 = 2')
+    >>> plt.plot(sol.t, sol.y[1], label='y0 = 4')
+    >>> plt.plot(sol.t, sol.y[2], label='y0 = 8')
+    >>> plt.legend()
+    >>> plt.xlabel('$t$')
```
Comment (nmayorov) [inline scipy/integrate/_ivp/ivp.py]: I suggest adding ylabel equal to '$y$'.

File: scipy/integrate/_ivp/ivp.py [hunk 2]
```diff
@@ -409,6 +417,14 @@ def solve_ivp(fun, t_span, y0, method='RK45', t_eval=None, dense_output=False,
     [[2.         1.21305369 0.73534021 0.27066736 0.01350938]
      [4.         2.42610739 1.47068043 0.54133472 0.02701876]
      [8.         4.85221478 2.94136085 1.08266944 0.05403753]]
+     
+    >>> import matplotlib.pyplot as plt
```
Comment (nmayorov) [inline scipy/integrate/_ivp/ivp.py]: Doesn't really matter, but this import should not be necessary.

**Comment by nmayorov** [comment]
Looks all right to me. I suggest that you should name the commit as "DOC: Add graphics to solve_ivp examples" and consider my minor remarks. I will merge after that.

File: scipy/integrate/_ivp/ivp.py (hunk 1 above)
Comment (jakobjakobson18) [reply scipy/integrate/_ivp/ivp.py]: Even though the property is not specified? If the example would explicitly refer to a concentration or any physical quantity I would have put it there in the first place. So I was unsure.

In the end it's up to you.

File: scipy/integrate/_ivp/ivp.py (hunk 2 above)
Comment (jakobjakobson18) [reply scipy/integrate/_ivp/ivp.py]: Took care of it.

File: scipy/integrate/_ivp/ivp.py (hunk 1 above)
Comment (nmayorov) [reply scipy/integrate/_ivp/ivp.py]: As `y` is used everywhere in the documentation of `solve_ivp` as the unknown function it makes sense to put this label.

**Comment by nmayorov** [comment]
Also update the master branch and rebase your branch on it to handle the build error.

**Comment by jakobjakobson18** [comment]
I'm closing this pull request and will open a new one as I don't know how to rebase my repository. So, thanks so far

**Comment by nmayorov** [comment]
It's not a good reason to close the PR. To rebase you need.

1. Checkout your local master branch.
2. Do `git pull upstream master`.
3. Switch back to your branch.
4. Do `rebase master`.

Go ahead, reopen and rebase.

**Comment by mdhaber** [comment]
@nmayorov He'll also have to force push to his repository after the rebase, right?
`git push origin <your_branch_name> --force`

**Comment by nmayorov** [comment]
Sure.

**Pull request closed without merging**